docs/HACKING.md

```diff
@@ -51,120 +49,106 @@ Add the following lines to your shell startup script (`.bashrc`, `.zshrc`, etc.)
 
-    $ workon goldstone-server
-
-This command will set your environment to use an isolated python version.  Running `deactivate` will revert to using the system python.  
+## Starting Goldstone Server
 
-The first time you enter the virtualenv, you should also install the project requirements, and some additional utilities.
+To start the development environment, execute:
 
-    $ pip install --upgrade pip
-    $ pip install -r requirements.txt
-    $ pip install -r test-requirements.txt
+    $ cd ~/devel/goldstone-server
+    $ ./bin/start_dev_env.sh
 
-If the requirements files change, you should rerun the `pip install` commands.
+The first time you start Golstone Server, it will probably take several minutes to download docker containers and perform configuraiton tasks.  You may see errors and missing data in the user interface. You may also see failures if you execute the test suite.  The data should be sufficiently populated in 10 minutes.  If you continue to see errors in the UI or in tests, [please submit an issue!](https://github.com/Solinea/goldstone-server/issues)
 
 
-## Building the Goldstone Containers
+## Stopping Goldstone Server
 
-All supporting services are available as docker containers. This step configures the Goldstone Server database, and is the final step before running the application.  It only needs to be done the first time you start Goldstone, and when you change ES or PostgreSQL schema.  You can rerun this step if you want to reinitialize, but realize that it will remove existing data in PostgreSQL and Elasticsearch.
+To stop the development environment, either:
 
-    $ cd $PROJECT_HOME/goldstone-server
-    $ bin/init_dev_env.sh
+* `<Ctrl-C>` from the window running `start_dev_env.sh`; or
+* run `stop_dev_env.sh` from another window. 
+ 
+This will stop both the docker VM and the OpenStack VM.
 
-Here is an [example of the output](https://gist.github.com/jxstanford/8abace13d7ed2d036e09#file-init_dev_env-out) from a successful run.
-If you prefer to configure your own OpenStack, you will need to follow the instructions for configuring OpenStack hosts in the [INSTALL](http://goldstone-server.readthedocs.org/en/latest/INSTALL/) guide.  You should also update your `postactivate` script to use proper values for the `OS_*` settings.
 
-## Starting/Stopping Goldstone Server
+## Configuring OpenStack Instance
 
-There are some convenience scripts in `$PROJECT_HOME/goldstone-server/bin` for starting and stopping the virtual machines and docker containers that support the goldstone test environment.  For developer flexibilty, starting/stopping the django application has been omitted from the scripts.  To start the development environment, execute:
+With the development environment started in another window, execute the following commands to configure the development OpenStack instance to ship logs and events back to Goldstone.  This only needs to be performed once:
```

Comment:
This sentence is correct if you strike, "With the development environment started in another window,"  The commands that follow start the development environment in another window.